src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+
+    private static final Map<String, Integer> FRUIT_KINDS_AND_QUANTITY = new HashMap<>();
+
+    public static Map<String, Integer> getFruitKindsAndQuantity() {
+        return FRUIT_KINDS_AND_QUANTITY;
+    }
```

Comment:
provide methods for put/get and possibly entrySet rather than simply returning the whole map in the getter

Reply:
remove this method